dashboard/app/models/level.rb

```diff
@@ -241,7 +241,9 @@ def to_xml(options = {})
     builder.to_xml(PRETTY_PRINT)
   end
 
-  PRETTY_PRINT = {save_with: Nokogiri::XML::Node::SaveOptions::NO_DECLARATION | Nokogiri::XML::Node::SaveOptions::FORMAT}
+  PRETTY_PRINT = {
+    save_with: Nokogiri::XML::Node::SaveOptions::NO_DECLARATION | Nokogiri::XML::Node::SaveOptions::FORMAT
+  }.freeze
```

Comment:
Apparently, calling `to_xml` on a hash mutates it (based on failing CircleCI test). Any opinion on the most sensical way to fix? Make this `@pretty_print`? Dup the object before converting to XML?